### ISSUE TITLE
Bufftrigger ignore opposing faction

### ISSUE THREAD

**Issue by jjholleman** [WeakAuras/WeakAuras2#4136]
Grouping with alliance/horde in open world makes you unable to buff them, however, when using a buff trigger to check if everyone in smartgroup has a buff, there is no way to ignore players from the opposing faction. 

Adding an _ignore other faction_ filter will solve this issue. 

https://i.imgur.com/SDynhD1.png

**Comment by jjholleman** [comment]
Hmm, that's true actually 🤔

**Comment by Luunae** [comment]
I'm here for the same buff-tracking reason.

I think the relevant thing to check here is hostility. I did some checking with a cross-faction party. Player A (Me) and Player B (Friend on the opposite faction).

To get debug info I setup a text display to show the Unit Characteristics of my Target (Trigger 1, for example). The notable Display Text is %1.hostility. When in an un-instanced area (We were in Dalaran, Broken Isles), if I targeted myself the Display output was "friendly". If I targeted my friend, the output was "hostile". We moved into an instance (Violet Hold), and then when I targeted her the output was "hostile".

Unfortunately I can't seem to figure out how to combine this info with the Smart Group, Raid, Party, etc. selections as is, but "Ignore Hostile" would be great!
